bin/cron.py

```diff
@@ -44,13 +44,15 @@
 MANAGE = str(ROOT_PATH / "manage.py")
 HEALTH_FILE_BASE = f"{DATA_PATH}/last-run"
 
+TIMEOUT = 60 * 9
```

Comment:
I'm guessing this means 9 minutes? Any reason you've chosen this amount of time? The default amount of time between task runs is 5 minutes I believe, so I think if it's going to take longer than that then we should just try again. I'd also appreciate a comment here indicating the purpose and length of time indicated, or at least whether this is in seconds, minutes, etc.

Reply:
Yeah, can't argue with any of that. It was a starting point for a discussion, and I agree that just under 5 min would be much better. 

Apols for the lack of context around the timeout, too 🤦